src/reducers/polling.js

```diff
@@ -312,7 +315,6 @@ export const pollsInit = () => async dispatch => {
               .service('smartContract')
               .getContract('POLLING').address;
             dispatch(addPoll(pollData));
-            dispatch(pollDataInit(pollData));
```

Comment:
previously, this is where we auto-fetched poll data for each poll returned from `getAllWhiteListedPolls`